format-specs/schema.json

```diff
@@ -25,20 +25,13 @@
               "enum": ["WKB"],
               "description": "Name of the geometry encoding format. Currently only 'WKB' is supported."
             },
-            "geometry_type": {
-              "oneOf": [
-                {
-                  "$ref": "#/$defs/geometry_type"
-                },
-                {
-                  "type": "array",
-                  "items": {
-                    "$ref": "#/$defs/geometry_type"
-                  },
-                  "uniqueItems": true
-                }
-              ],
-              "description": "The geometry type(s) of all geometries, or 'Unknown' if they are not known."
+            "geometry_types": {
+              "description": "The geometry types of all geometries, or an empty array if they are not known.",
+              "type": "array",
```

Comment:
I'd propose to add "uniqueItems" set to `true` so that it can't be `["Point", "Point"]` :-)

Reply:
Good point.  Added in the latest commit.